sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -367,15 +330,34 @@ def __str__(self):
                 ('u' if self.update else '') +
                 ('p' if self.process else ''))
 
-
-AccountPermissions.READ = AccountPermissions(read=True)
-AccountPermissions.WRITE = AccountPermissions(write=True)
-AccountPermissions.DELETE = AccountPermissions(delete=True)
-AccountPermissions.LIST = AccountPermissions(list=True)
-AccountPermissions.ADD = AccountPermissions(add=True)
-AccountPermissions.CREATE = AccountPermissions(create=True)
-AccountPermissions.UPDATE = AccountPermissions(update=True)
-AccountPermissions.PROCESS = AccountPermissions(process=True)
+    @classmethod
+    def from_string(cls, permission):
+        if len(permission) > 8:
+            raise ValueError("Invalid Permission String")
+        read, write, delete, list = False, False, False, False
```

Comment:
`list` is a reserved word that we should avoid redefining.
Maybe use names like: `p_read, p_write, p_list...`